commands.go

```diff
@@ -1367,6 +1367,18 @@ func (c *commandable) ClientPause(dur time.Duration) *BoolCmd {
 	return cmd
 }
 
+func (c *commandable) SetName(name string) *StatusCmd {
+	cmd := NewStatusCmd("CLIENT", "SETNAME", name)
```

Comment:
You can also use `NewBoolCmd` here and the result will be true if Redis returns "OK". But I am fine with `StatusCmd` too.